examples/pulse.hlsl

```diff
@@ -0,0 +1,35 @@
+//original by Charles 'Surn' Fettinger for obs-shaderfilter
+//Converted and made linux compatible by John Jerome 'Wishdream' Romero for obs-shaderfilter-plus 10/2020
+//uv_scale and uv_offset aren't applied/ignored entirely from original
+
+#pragma shaderfilter set speed__description Speed
+#pragma shaderfilter set min_growth_pixels__description Min Growth Pixels
+#pragma shaderfilter set max_growth_pixels__description Max Growth Pixels
+
+uniform float speed;
+uniform float min_growth_pixels;
+uniform float max_growth_pixels;
```

Comment:
Default values should be specified such that it is obvious that the shader is doing something when added.

Reply:
Completely forgot to place one!
Should be fixed in https://github.com/Wishdream/obs-shaderfilter-plus/commit/9eef8197fe3396d13e4e24756d6fd5a31bfd5fdb